### DISCUSSION TITLE
Fresh install unable to load port 4000

### DISCUSSION THREAD

**Discussion by mikec-design** [teslamate-org/teslamate#2617] [Q&A]
I've nearly given up on getting Teslamate back up and running.

I've created a fresh install on a Raspberry Pi 3 Model B Rev 1.2 running Docker version 20.10.14, build a224086. From what I can recall the install went off without a hitch. No errors were reported but I wasn't able to load the config on port 4000. Chromium reported that the site could not be reached. I am able to load port 3000 and get to Grafana's login screen.

Based on https://github.com/adriankumpf/teslamate/issues/2302 I tried upgrading libseccomp2 (https://blog.samcater.com/fix-workaround-rpi4-docker-libseccomp2-docker-20/)

I initially installed the latest and then downgraded to 1.2.5 as suggested in the issue linked above. But I still cannot access port 4000. Any suggestions would be really appreciated! I really miss having Teslamate's data.

My docker-compose.yml file is below:

`version: "3"

services:
  teslamate:
    image: teslamate/teslamate:1.25.0
    restart: always
    environment:
      - ENCRYPTION_KEY= keyname
      - DATABASE_USER=username
      - DATABASE_PASS= password
      - DATABASE_NAME=teslamate
      - DATABASE_HOST=database
      - MQTT_HOST=mosquitto
    ports:
      - 4000:4000
    volumes:
      - ./import:/opt/app/import
    cap_drop:
      - all

  database:
    image: postgres:13
    restart: always
    environment:
      - POSTGRES_USER=username
      - POSTGRES_PASSWORD=password
      - POSTGRES_DB=teslamate
    volumes:
      - teslamate-db:/var/lib/postgresql/data

  grafana:
    image: teslamate/grafana:latest
    restart: always
    environment:
      - DATABASE_USER=username
      - DATABASE_PASS=password
      - DATABASE_NAME=teslamate
      - DATABASE_HOST=database
    ports:
      - 3000:3000
    volumes:
      - teslamate-grafana-data:/var/lib/grafana

  mosquitto:
    image: eclipse-mosquitto:2
    restart: always
    command: mosquitto -c /mosquitto-no-auth.conf
    # ports:
    #   - 1883:1883
    volumes:
      - mosquitto-conf:/mosquitto/config
      - mosquitto-data:/mosquitto/data

volumes:
  teslamate-db:
  teslamate-grafana-data:
  mosquitto-conf:
  mosquitto-data:
`

**Comment by cwanja** [comment]
Upgrade to Bullseye? Technically if you are 1.25, it should work on Buster with or without the libseccomp2 backport.
